ts/sagas/startup.ts

```diff
@@ -436,10 +453,14 @@ export function* initializeApplicationSaga(
   yield* fork(
     watchCheckSessionSaga,
     backendClient.getSession,
-    backendClient.getSupportToken
+    backendClient.getSupportToken,
+    formatRequestedTokenString(needRefreshZendeskToken)
   );
-
+  yield* put(zendeskTokenNeedsRefresh(false));
```

Comment:
Is that needed to "reset" a previous `zendeskToken` refresh handling? We cannot handle the reset outside the startup saga?